apps/smartling/frontend/src/Sidebar.tsx

```diff
@@ -204,7 +207,12 @@ export default class Sidebar extends React.Component<Props, State> {
     const { sdk, projectId } = this.props;
     const { token } = this.state;
 
-    const res = await smartlingClient.getLinkedJobs(token, sdk.ids.space, projectId, sdk.ids.entry);
+    const res = await smartlingClient.getLinkedJobs(
```

Comment:
Not sure which is the better solution there. But what do you think of:
```suggestion
    if (token === null) {
      this.runAuthFlow(true);
      return;
   }
 
    const res = await smartlingClient.getLinkedJobs(
```
This avoids sending a request if the token is null'd, and avoids the need for type casting token to string